webhook/webhook.go

```diff
@@ -0,0 +1,102 @@
+package webhook
+
+import (
+	"bytes"
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	log "github.com/gophish/gophish/logger"
+)
+
+const (
+
+	//DefaultTimeoutSeconds is amount of seconds of timeout used by HTTP sender
```

Comment:
Minor nit - all of these should be `// Comment`, not `//Comment` (note the space). I understand there are a couple of instances in the codebase where I make the same mistake, but best to fix it now rather than having to go back and fix it later.

Reply:
fixed